### PR TITLE
vlc: playing video from the web

### PR DESCRIPTION
**Pull request by Kreyren**
- [X] The page (if new), does not already exist in the repo.
	- Exists
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
	- Updating already existing page
- [X] The page has 8 or fewer examples.
	- As of 29.06.2020 page has 4 examples, 5 with mine
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
	- Concluded that it does follow these guidelines
- [X] The page description includes a link to documentation or a homepage (if applicable).
	- Already present

### CODE REVIEW

**Comment by Kreyren** [comment]
CLA ain't working on my end

image

and i am not really interested in signing one~

**Comment by Kreyren** [comment]
The `TLDR005` seems irelevant to this issue assuming that i wasn't touching description

**Comment by Kreyren** [comment]
GitHub CI: `We are currently unable to download the log. Please try again later.`

**Comment by Kreyren** [comment]
Due to the https://github.com/tldr-pages/tldr/pull/4136#issuecomment-651317857 addressed CLA in [https://github.com/tldr-pages/tldr/pull/4136/commits/bfbe5703ba439fe52cd1c9753bc2cd56cef780e7 ](https://github.com/tldr-pages/tldr/pull/4136/commits/bfbe5703ba439fe52cd1c9753bc2cd56cef780e7) see valid signature and commit message

File: pages/common/vlc.md [hunk 1]
```diff
@@ -18,3 +18,7 @@
 - Play repeatedly:
 
 `vlc --loop {{path/to/file}}`
+
+- Play video from the web:
+
+`vlc https://www.youtube.com/watch?v=oHg5SJYRHA0`
```
Comment (zdroid) [inline pages/common/vlc.md:24]:
```suggestion
- Play video from a URL:

`vlc {{https://www.youtube.com/watch?v=oHg5SJYRHA0}}`
```
I think this is more clear. Braces around the URL are there to show that the URL can be changed (i.e. you can put another YouTube link in the same place).

**Comment by zdroid** [comment]
@Kreyren Very weird about CLA. Can you try again?

**Comment by Kreyren** [comment]
> @Kreyren Very weird about CLA. Can you try again?

Same issue, tried to unblock script and fetch for 
`cloudflare.com` and `cla-assistant.io` nothing else seems to be requested for noscript.

image

Maybe blocking tor users? The commit should be enough for a CLA though

**Comment by Kreyren** [comment]
Or like i can make custom CLA and sign it using gpg if you want

**Comment by zdroid** [comment]
It must be an issue with your browser (settings). Do you have JavaScript or images blocked?

**Comment by Kreyren** [comment]
> Do you have JavaScript or images blocked? @zdroid

I had, allowed them for the mensioned domains which didn't affect the issue too much (just the `Hey!` dissapeared)

**Comment by zdroid** [comment]
@Kreyren I think you need to allow it for github.com too (it uses GitHub API), and maybe for some other. So you know, it's problematic if JS is blocked in your browser.

**Comment by Kreyren** [comment]
Made it to work

image


it needed `fetch()`

**Pull request merged**